src/healthkit.cs

```diff
@@ -2285,6 +2285,18 @@ enum HKQuantityTypeIdentifier {
 		[Watch (9, 0), MacCatalyst (16, 0), Mac (13, 0), iOS (16, 0), NoTV]
 		[Field ("HKQuantityTypeIdentifierAtrialFibrillationBurden")]
 		AtrialFibrillationBurden,
+
+		[Watch (9, 0), MacCatalyst (16, 0), Mac (13, 0), iOS (16, 0)]
+		[Field ("HKQuantityTypeIdentifierAppleSleepingWristTemperature")]
+		AppleSleepingWristTemperature,
```

Comment:
From what I remember before, AppleSleepingWristTemperature, UnderwaterDepth, and WaterTemperature were actually only available on Watch and maybe iOS and failed in the tests that aren't done running at the time of this comment. Maybe this has changed since Xcode14.0 rc though?